packages/manager/modules/anthos/src/dashboard/access-restriction/table/controller.js

```diff
@@ -1,4 +1,5 @@
 import IpBlock from './ipblock.class';
+import { TRACKING_CHUNK } from '../constants';
```

Comment:
To avoid relying on constants located in parent folder, maybe you could expose the constant in routing (For example at packages/manager/modules/anthos/src/dashboard/routing.js)

Reply:
Moved into its own constant file